docs/troubleshoot/index.md

```diff
@@ -36,7 +36,7 @@ Review the following questions when you encounter an issue with {{site.data.keyw
 - [Troubleshooting node errors](#troubleshooting_node_errors)
 - [How to uninstall Podman on RHEL?](#uninstall_podman)
 - [Are you encountering HTTP error, while executing deploy-mgmt-hub.sh?](#deploy_mgmt_http_error)
-
+- [How documentation are sourced across repositories and how to correctly modify them?](#docs_source_explanation)
```

Comment:
I agree with the addition of these GHA actions details in the top level open-horizon.github.io repo README. I like the details there.

**How** we construct the documentation is NOT interesting inside the documentation.  We don't need this section in the troubleshooting guide.  Let's remove this addition to troubleshooting/index.md

Reply:
addressed